livy/src/main/java/org/apache/zeppelin/livy/LivySparkInterpreter.java

```diff
@@ -38,13 +38,20 @@
   private LivyOutputStream out;
 
   protected static Map<String, Integer> userSessionMap;
+  protected static Map<Integer, String> sessionId2AppIdMap;
+  protected static Map<Integer, String> sessionId2WebUIMap;
+
   private LivyHelper livyHelper;
+  private boolean displayAppInfo;
 
   public LivySparkInterpreter(Properties property) {
     super(property);
     userSessionMap = new HashMap<>();
+    sessionId2AppIdMap = new HashMap<>();
+    sessionId2WebUIMap = new HashMap<>();
     livyHelper = new LivyHelper(property);
     out = new LivyOutputStream();
+    this.displayAppInfo = Boolean.parseBoolean(getProperty("zeppelin.livy.displayAppInfo"));
```

Comment:
This key is different from `livy/src/main/resources/interpreter-setting.json` i.e. livy.spark.displayAppInfo